packages/apollo/src/git.ts

```diff
@@ -34,22 +35,22 @@ export interface GitContext {
   branch?: string;
 }
 
-export const gitInfo = async (): Promise<GitContext | undefined> => {
+export const gitInfo = async (
+  log: Command["log"]
```

Comment:
Might want the type of log to be something a bit more generic, since we're passing in `console.log` for testing.

Paraphrasing and best guessing, but something like this?
`logger: (...args: string[]) => void`

This is a non-blocking comment.

Reply:
hmm they seem to have the same sort of signature `log(message?: string, ...args: any[]): void;` and I think we want to know when the types of the Command.log become incompatible